doc/reference/dune-workspace/pin.rst

```diff
@@ -0,0 +1,46 @@
+pin
+===
+
+.. warning::
+
+   Dune Package Management is not final yet and the configuration options
+   are subject to change.
+
+This stanza is used to define additional package sources.
```

Comment:
:art: I would suggest the same as the other two: link to package management and a description that this is related to package management apart from within the warning.

Again, feel free to ignore if you don't think this is necessary :smile:

Reply:
I've changed it a bit to mention that :+1: